### DISCUSSION TITLE
NASA Openscapes 2i2c Clean-up

### DISCUSSION THREAD

**Discussion by erinmr** [Openscapes/openscapes.cloud#44] [Announcements]
Hi @NASA-Openscapes/removeaccess-2i2c Team - This team was formed b/c I didn't have a good way to reach you by email. From what we could tell you haven't accessed the hub in over six months. We will be deleting your home directories on Monday, November 27. I wanted to let you know in case there is any work you'd like to download. If you have any questions, you can reply to this thread or email me (erin @ metadatagamechangers.com). Thanks and apologies for this hacky way to get in touch. Best - Erin

**Comment by andypbarrett** [comment]
Hi Erin

Looks like I ended up in the remove access group for 2i2c.  I would like to
retain access to the hub.  If that is OK.

Cheers

Andy

**Reply by erinmr** [reply]
Hi @andypbarrett - I have you on the long-term access team. All ok :-)

**Comment by eeholmes** [comment]
Hi Erin, Looks like I might have ended up on the removeaccess team also. I have definitely been on in the last 6 months and would like to retain access, if that's ok. --Eli

**Comment by erinmr** [comment]
I think everyone who is associated with the repo got this message, instead
of just notifying the remove team. I thought if I sent this and tagged the
remove team, only they would be notified. :-/ I am sorry. (Eli - you are on
the long-term team. )

**Reply by MattF-NSIDC** [reply]
I'm also on long-term if I remember correctly and got notified. I think it's because I have this repo set to watch "all activity". I don't know if there's a way to _prevent_ notifying watchers... :thinking:

**Comment by biplovbhandari** [comment]
Hi Erin, Looks like I might have ended up on the removeaccess team also. I would like to retain access, if that's ok. - Biplov

**Reply by erinmr** [reply]
I have moved you over to the long-term team. Thanks!

**Comment by DhruvaKathuria** [comment]
Hi Erin,

I got the access for the team around 2 weeks back and I would definitely
love to explore more and collaborate with the team. If possible, I would
like to retain access as well.

Dhruva

On Mon, Nov 13, 2023 at 11:20 AM Erin Robinson ***@***.***>
wrote:

> I have moved you over to the long-term team. Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NASA-Openscapes/2i2cAccessPolicies/discussions/4#discussioncomment-7556005>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIUIZHVBYXFZYTOBNWDNJB3YEJCEDAVCNFSM6AAAAAA7HK6RV6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TKNJWGAYDK>
> .
> You are receiving this because you are on a team that was mentioned.Message
> ID:
> <NASA-Openscapes/2i2cAccessPolicies/repo-discussions/4/comments/7556005@
> github.com>
>


-- 
Dhruva Kathuria

**Reply by erinmr** [reply]
Hi Dhruva - I made the switch for you :-) Thanks!

**Comment by drsudhanshupanda** [comment]
Erin, please make me retain the access. ECOSTRESS project just rolled and I will use it more.
Regards,
Sudhanshu

Sent from my T-Mobile 4G LTE Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by erinmr** [reply]
Hi Sudhanshu - Please fill out this form -> https://forms.gle/xtxWQTbys1U2BzocA so that I can share details on access with you. Best, Erin

**Comment by johnwilkin** [comment]
Hey folks,

I don't think the whole list needs to be in cc on all these emails (please!)

Thanks, JOhn.



--
John Wilkin - Rutgers University Ocean Modeling Group - tel: (g-voice): +1-609-630-0559

This email was sent at a time that is convenient for me. I do not expect you to respond to it outside of your own usual working hours.

________________________________

**Reply by yuvipanda** [reply]
@johnwilkin this conversation is happening on github, so if you're getting an email for every reply here you can either come to the github link and unsubscribe from this specific conversation, or (my recommendation) adjust your github notifications to not send you an email for every notification on github! https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications has more information on how you can do the latter.

**Comment by brianmapes** [comment]
Hi Erin.

Actually, student Snigdha and I had just discovered the inefficiencies of our laptop -opendap data processing.

We might hope to still do a Dask-enhanced cloud processing! Can there be an extension?

Ha ha, email being “hacky” nowadays.

Thanks,
Brian

On Nov 11, 2023, at 11:00 AM, Erin Robinson ***@***.***> wrote:

﻿
CAUTION: This email originated from outside the organization. DO NOT CLICK ON LINKS or OPEN ATTACHMENTS unless you know and trust the sender.

Hi @NASA-Openscapes/removeaccess-2i2c<https://urldefense.com/v3/__https://github.com/orgs/NASA-Openscapes/teams/removeaccess-2i2c__;!!KVu0SnhVq1hAFvslES2Y!JvV0fb73vUl9d-lF07CgQ4GxGp9b4rIt_4Xe_UysAxH2EPYAphOMbY4aWL6Zoy6NA_tbvGj59iSfDfl6ngI_7HM$> Team - This team was formed b/c I didn't have a good way to reach you by email. From what we could tell you haven't accessed the hub in over six months. We will be deleting your home directories on Monday, November 27. I wanted to let you know in case there is any work you'd like to download. If you have any questions, you can reply to this thread or email me (erin @ metadatagamechangers.com). Thanks and apologies for this hacky way to get in touch. Best - Erin

**Comment by JessicaS11** [comment]
Hey @erinmr I think this message and my request for access for this week's tutorial must have crossed paths. I'd like to retain access for ease of ongoing collaborations with the OpenScapes DAAC folks, if that's possible. Thanks!
